### PR TITLE
Added Stats Section

### PR DESCRIPTION
**Pull request by arujjval**
-> Added Stats section
-> modified index.html and vite.config.js (as they were causing issues with rendering)

### CODE REVIEW

File: src/Sections/Home.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ export const Home = () => {
         <div className="w-1/3">
           <TextGenerateEffect
             className="mt-[300px]"
-            words="&quot;DevOps isn't just about breaking down silos; it's about breaking down barriers, fostering collaboration, and enabling teams to build and deliver amazing things together.&quot;"
+            words="&quot;Dev isn't just about breaking down silos; it's about breaking down barriers, fostering collaboration, and enabling teams to build and deliver amazing things together.&quot;"
```
Comment (satyampsoni) [inline src/Sections/Home.tsx]: Why this change?
Done by mistake ??

File: vite.config.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ export default defineConfig({
       "@": path.resolve(__dirname, "./src"),
     },
   },
-  build: {
-    outDir: '.', 
-  },
+  // build: {
```
Comment (satyampsoni) [inline vite.config.ts]: Please remove this as of now!! 
I'll look into it later

**Comment by satyampsoni** [comment]
Cc @Saksham1387

File: src/Sections/Home.tsx (hunk 1 above)
Comment (arujjval) [reply src/Sections/Home.tsx]: yeah, its a mistake.

**Comment by arujjval** [comment]
@satyampsoni 
Should I close this PR and make a new one?
